# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1518088744 -3600
#      Thu Feb 08 12:19:04 2018 +0100
# Node ID 80ad1a9de879a3c2549a127b66d8d1ad920d8a13
# Parent  30e1d0bf7ade11f308bdc915e3b2d509166c8dc7
* varaible-editor.h: make tabs closable and movable again

diff --git a/libgui/src/variable-editor.cc b/libgui/src/variable-editor.cc
--- a/libgui/src/variable-editor.cc
+++ b/libgui/src/variable-editor.cc
@@ -117,23 +117,23 @@ var_editor_tab::set_editable (bool edita
 
 // Functions for reimplemented tab widget.
 
 var_editor_tab_widget::var_editor_tab_widget (QWidget *p)
   : QTabWidget (p)
 {
   tab_bar *bar = new tab_bar (this);
 
-  setTabsClosable (true);
-  setMovable (true);
-
   connect (bar, SIGNAL (close_current_tab_signal (bool)),
            p->parent (), SLOT (request_close_tab (bool)));
 
   this->setTabBar (bar);
+
+  setTabsClosable (true);
+  setMovable (true);
 }
 
 tab_bar *
 var_editor_tab_widget::get_tab_bar (void) const
 {
   return qobject_cast<tab_bar *> (tabBar ());
 }
 
